presto-main/src/main/java/com/facebook/presto/operator/WindowOperator.java

```diff
@@ -192,6 +192,7 @@ public OperatorFactory duplicate()
     private final PageBuilder pageBuilder;
 
     private final SingleDriverWindowInfoBuilder windowInfo;
+    private SingleDriverWindowInfo singleDriverWindowInfo;
```

Comment:
Removed reference to `HashAggregationOperator` and added one line of explanation (0af2be44bc78a40530c28c95084eabb6c84597f0).